maro/simulator/core.py

```diff
@@ -21,37 +20,42 @@ class Env(AbsEnv):
     """Default environment implementation using generator.
 
     Args:
-        scenario (str): Scenario name under maro/sim/scenarios folder.
-        topology (str): Topology name under specified scenario folder,
-            if this point to a existing folder, then it will use this as topology for built-in scenario.
+        scenario (str): Scenario name under maro/simulator/scenarios folder.
+        topology (str): Topology name under specified scenario folder.
+            If this points to an existing folder, the corresponding topology will be used for the built-in scenario.
         start_tick (int): Start tick of the scenario, usually used for pre-processed data streaming.
         durations (int): Duration ticks of this environment from start_tick.
         snapshot_resolution (int): How many ticks will take a snapshot.
-        max_snapshots(int): Max in-memory snapshot number, default None means keep all snapshots in memory,
-            when taking a snapshot, if it reaches this limitation, oldest one will be overwrote.
-        business_engine_cls : Class of business engine, if specified, then use it to construct be instance,
-            or will search internal by scenario.
+        max_snapshots(int): Max in-memory snapshot number.
+            When taking a snapshot, if it reaches this limitation, oldest one will be overwrote.
+            None means keeping all snapshots in memory. Defaults to None.
+        business_engine_cls : Class of business engine. If specified, use it to construct the be instance,
```

Comment:
It will be used to construct the business engine instance.

Confirm this part with  @chaosddp  `or serach...` , it should be removed or not?

Reply:
Oh, I see, "If it was not specified, we will construct the business engine instance based on the scenario name."